stanify/builders/stan_model.py

```diff
@@ -0,0 +1,390 @@
+                raise Exception("init_state may be set to True only for stock variables.")
+            self.stan_model_context.sample_statements.append(SamplingStatement(f"{variable_name}_init", distribution_type, *args, lower=lower, upper=upper, init_state=init_state))
+            self.stan_model_context.all_stan_variables.add(f"{variable_name}_init")
+        else:
+            variable_name = vensim_name_to_identifier(variable_name)
+            self.stan_model_context.all_stan_variables.add(variable_name)
+            for arg in args:
+                if isinstance(arg, str):
+                    # If the distribution argument is an expression, parse the dependant variables
+                    # We're using the python parser here, which might be problematic
+                    used_variable_names = [vensim_name_to_identifier(node.id.strip()) for node in ast.walk(ast.parse(arg)) if isinstance(node, ast.Name)]
+                    for name in used_variable_names:
+                        if vensim_name_to_identifier(name) not in set(self.vensim_model_context.variable_names).union(set(self.stan_model_context.all_stan_variables)):
+                            sample_string = f"{variable_name} ~ {distribution_type}({', '.join(args)})"
+                            raise Exception(f"{sample_string} : '{name}' doesn't exist in the Vensim model or the Stan model!")
+                        if name in self.vensim_model_context.variable_names and name not in self.vensim_model_context.integ_outcome_vector_names:
+                            self.stan_model_context.exposed_parameters.update(used_variable_names)
+
+            if variable_name in self.vensim_model_context.variable_names and variable_name not in self.vensim_model_context.integ_outcome_vector_names:
+                self.stan_model_context.exposed_parameters.add(variable_name)
+            self.stan_model_context.sample_statements.append(SamplingStatement(variable_name, distribution_type, *args, lower=lower, upper=upper, init_state=init_state))
+
+
+    def set_type(self, est_param_names: list, hier_est_param_names: list, target_simulated_vector_names: list, driving_vector_names: list, model_name: str):
+        self.est_param_names = est_param_names
+        self.hier_est_param_names = hier_est_param_names
+        # TODO @Dashadower how to make target_simulated_vector_names and integ_outcome_vector_names consistent? make class TypeContext? related to lin
+        self.stan_model_context.target_integ_outcome_vector_names = target_simulated_vector_names
+        self.stan_model_context.obs_integ_outcome_vector_names = [f'{name}_obs' for name in self.stan_model_context.target_integ_outcome_vector_names]
+        self.driving_vector_names = driving_vector_names
+        self.model_name = model_name
+
+    #TODO @dashadower for external refernce of target_simulated_vector_names
+    # is it better to use model.target_simulated_vector_names or define function for consistency?
```

Comment:
> for external refernce of target_simulated_vector_names is it better to use model.target_simulated_vector_names or define function for consistency?

Since `target_integ_outcome_vector_names` is a class attribute of `self.stan_model_context`, I would make a function return it instead of having to use attribute access operator(`.`) twice.